src/main/state/encrypt/index.js

```diff
@@ -0,0 +1,84 @@
+/** From http://lollyrock.com/articles/nodejs-encryption/. */
+
+const crypto = require('crypto');
+
+const fse = require('fs-extra');
+
+const ALGORITHM = 'aes-256-cbc';
+
+/**
+ * Encrypt the state with the given key.
+ *
+ * @param {Object} state - State tree.
+ * @param {string} key - Key to encrypt, as an utf8 string.
+ * @returns {string} The state encrypted, as an utf8 string.
```

Comment:
Don't we have a jsdoc rule imposing an hyphen between the parameter type and the description ?

Reply:
Like that ?
```js
/**
 * @returns {string} - The state encrypted, as an utf8 string.
 */
```

I don't think so, only between the param name and its description.

Reply:
Yup, like that. Should we ?

Reply:
I don't think so 😄